app/charts/line/lines-state.tsx

```diff
@@ -132,12 +133,18 @@ const useLinesState = (
   const xScaleTimeRange = scaleTime().domain(xScaleTimeRangeDomain);
 
   // y
-  const minValue = Math.min(min(scalesData, getY) ?? 0, 0);
+  const minValue = getMaybeDynamicMinYScaleValue(
+    yMeasure.scaleType,
+    min(scalesData, getY)
+  );
   const maxValue = max(scalesData, getY) ?? 0;
   const yDomain = [minValue, maxValue];
   const yScale = scaleLinear().domain(yDomain).nice();
 
-  const paddingMinValue = Math.min(min(paddingData, getY) ?? 0, 0);
+  const paddingMinValue = getMaybeDynamicMinYScaleValue(
+    yMeasure.scaleType,
+    min(paddingData, getY)
+  );
```

Comment:
Would it make sense to use a hook like useYScale similar to the useYScales used in he combo charts ?